src/clap.rs

```diff
@@ -237,14 +238,14 @@ pub fn build_app() -> Command<'static> {
         .long("color")
         .takes_value(true)
         .value_name("COLOR SCHEME")
-        .possible_values([
+        .value_parser(PossibleValuesParser::new(vec![
```

Comment:
nit: no need for `vec!` here, a bare array should be fine:

```rust
PossibleValuesParser::new([
            "default",
            "default-light",
            "gruvbox",
            "gruvbox-light",
            "nord",
            "nord-light",
        ]
```

Reply:
updated